### PR TITLE
Add links to the examples

### PR DESCRIPTION
**Pull request by violetagg**
- Remove unnecessary attributes
- Add global attribute for the source type

### CODE REVIEW

File: docs/modules/ROOT/pages/http-client.adoc [hunk 1]
```diff
@@ -1,4 +1,7 @@
+:source-language: java
```
Comment (pderop) [inline docs/modules/ROOT/pages/http-client.adoc:1]: what do you think about moving the source-language attribute to antora.yml ?

File: docs/modules/ROOT/pages/http-server.adoc [hunk 2]
```diff
@@ -1,4 +1,7 @@
+:source-language: java
```
Comment (pderop) [inline docs/modules/ROOT/pages/http-server.adoc:1]: what do you think about moving the source-language attribute to antora.yml ?

File: docs/modules/ROOT/pages/tcp-client.adoc [hunk 3]
```diff
@@ -1,4 +1,7 @@
+:source-language: java
```
Comment (pderop) [inline docs/modules/ROOT/pages/tcp-client.adoc:1]: what do you think about moving the source-language attribute to antora.yml ?

File: docs/modules/ROOT/pages/tcp-server.adoc [hunk 4]
```diff
@@ -1,9 +1,11 @@
+:source-language: java
```
Comment (pderop) [inline docs/modules/ROOT/pages/tcp-server.adoc:1]: what do you think about moving the source-language attribute to antora.yml ?

File: docs/modules/ROOT/pages/udp-client.adoc [hunk 5]
```diff
@@ -1,4 +1,7 @@
+:source-language: java
```
Comment (pderop) [inline docs/modules/ROOT/pages/udp-client.adoc:1]: what do you think about moving the source-language attribute to antora.yml ?

File: docs/modules/ROOT/pages/udp-server.adoc [hunk 6]
```diff
@@ -1,4 +1,7 @@
+:source-language: java
```
Comment (pderop) [inline docs/modules/ROOT/pages/udp-server.adoc:1]: what do you think about moving the source-language attribute to antora.yml ?

**Review by pderop — APPROVED** [review]
+1
(see few comments, I'm proposing to move the source-language attribute to antora.yml)

**Comment by violetagg** [comment]
> +1 (see few comments, I'm proposing to move the source-language attribute to antora.yml)

I moved also several other attribute in antora.yml
Please check again

**Review by pderop — APPROVED** [review]
nice !

**Comment by violetagg** [comment]
The failing CI is not related as this change is only related to the documentation.

**Pull request merged**